campaigns/tdc/15_the_doom_of_arkham_part_1.json

```diff
@@ -548,6 +656,22 @@
         ]
       }
     },
+    {
+      "id": "earn_xp_bonus",
+      "text": "Each investigator earns experience equal to the Victory X value of each card in the victory display. (The bonus experience granted by the act will be included automatically.)",
```

Comment:
We should put this string into the `confirm_text` inside the input.  It will have the 'included automatically' paren automatically.